app/controllers/job_post_requests_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
       )
       @job.user = @user
       @job.save
-      SystemMailer.job_post_request(@job_post_request, @job).deliver
+      SystemMailer.job_post_request(@job_post_request, @job).deliver_now
```

Comment:
I'm not 100% sure how this was working before as I couldn't find the implementation of a `#deliver` method in action mailer's code...